installation/preparing-to-install/en.md

```diff
@@ -14,22 +14,40 @@ You can download a Solus ISO by going to our [Download page](/download).
 
 ### Linux and Mac
 
-Download the relevant Solus ISO SHA256SUMS: File, Signed File and Public Key from the [Download page](/download).
+Download the relevant Solus ISO SHA256SUMS: File and Signed File from the [Download page](/download).
 
-**Note:** It is advisable that the files are all placed in the same folder.
+#### Obtain the Public Key
 
-#### Import the Public Key
+The public GPG key can be obtained from the [Download page](/download). Afterwards it can be imported with
 
 ```bash
 gpg --import <Public Key>.gpg
 ```
 
+However, it is generally advisable to obtain the key from a separate server. The reason for this is that if the Solus website was ever compromised, an attacker replacing the Solus ISO with their own tampered version could easily also replace the key with their own. The below example locates and imports the key from the SKS keyservers (type the key number when prompted to import it).
```

Comment:
By that logic they could also replace the Help Center article and change the instructions to point to another key. None of this is really necessary IMO and is just security theatre.

Reply:
True. My point was more about raising awareness I guess. You could also argue that hosting the ISO, GPG key and instructions on the same server is security theater just as much because an attacker would gain control to change all three if the server was ever compromised. This happened with Mint back in 2016 but I think they were not signing their ISOs then. Anyway, point taken as this might be beyond the scope of what Solus is meant to be.